### ISSUE TITLE
Pattern does not have way to view whole if cut off

### ISSUE THREAD

**Issue by imbstack** [taskcluster/taskcluster#1455]
**Describe the bug**
In the schema viewers of the docs site, if a string has a pattern that is too long, it will be cut off and an ellipsis will be shown. This is great, the only issue is that there's no way I can find to see the entire thing

**To Reproduce**
1. Go to [the listTaskGroup api doc](https://docs.taskcluster.net/docs/reference/platform/queue/api#listTaskGroup)
2. Scroll down to `taskGroupId` in the Response Payload
4. See that `^[A-Za-z0-9_-]{...` has no way to expand to the entire thing.

**Expected behavior**
It would be nice if there was a tooltip on hover or something of the sort.

**Screenshots**

![image](https://user-images.githubusercontent.com/127521/66073616-431f3e00-e50c-11e9-8ee9-b11ce505ebbd.png)

**Platform (please complete the following information):**
 - OS: ubuntu
 - Browser: firefox
 - Version: 71.0a1 (2019-10-01) (64-bit)

**Comment by Shubhamchinda** [comment]
I'd like to take this!

**Comment by imbstack** [comment]
I believe the fix will occur in https://github.com/taskcluster/react-schema-viewer

**Comment by djmitche** [comment]
Hi! Sorry @Shubhamchinda but this issue is reserved for [outreachy](https://www.outreachy.org/) candidates.

Outreachy applicants: in order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

**Comment by Shubhamchinda** [comment]
> Hi! Sorry @Shubhamchinda but this issue is reserved for [outreachy](https://www.outreachy.org/) candidates.

No problem @djmitche . I'll find another issue, maybe not a `good first issue` as I want to explore more. Let me know if there's anything I can work on! 
Cheers!

**Comment by Ivilili** [comment]
Hi! I'm Outreachy candidate, can I work on this one?

**Comment by djmitche** [comment]
You can!

**Comment by djmitche** [comment]
@Ivilili are you still working on this?

**Comment by Ivilili** [comment]
Yes.. still working on this

**Comment by Ivilili** [comment]
Hi @djmitche @helfi92 I have a question about this issue. It's not only taskGroupId pattern that is cut, but also every other longer pattern. Should I make a pattern component, so that every longer pattern has a tooltip?

**Comment by helfi92** [comment]
When I run https://github.com/taskcluster/react-schema-viewer locally, and then navigate to http://localhost:5005/, I am able to see the title in a Bootstrap tooltip.

<img width="399" alt="Screen Shot 2019-10-08 at 3 33 47 PM" src="https://user-images.githubusercontent.com/3766511/66426967-0dcd9100-e9e1-11e9-9400-25fdfb10ab59.png">

For some reason it's not showing up when we use it in the Taskcluster site. We should figure out why and send a fix. Does that help unblock you?

**Comment by helfi92** [comment]
@Ivilili Are you still working on this issue?

**Comment by Ivilili** [comment]
@helfi92 sorry, I got stuck on this one for too long. The solution is probably simple...someone else can take this one.

**Comment by Cressence** [comment]
Hello i am an outreachy applicant, can I work on this?

**Comment by helfi92** [comment]
@Cressence In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema). Thanks!

**Comment by Vane237** [comment]
Hello! I am Vanessa an outreachy applicant, can I work on this?

**Comment by helfi92** [comment]
How is it going @Vane237?

**Comment by Vane237** [comment]
While checking the code to know what is happening, I actually noticed the popover displaying on hover but it is displayed out of the page view port (probably why it can't be seen).
I tried writing some style to override the positioning of the popup but no happy result so far..
Still trying though

**Comment by helfi92** [comment]
Interesting. This makes me think that we might need to adjust the `z-index` property of the tooltip to be similar to material-ui's tooltip z-index which seems to be `1500` when looking at the [default theme](https://material-ui.com/customization/default-theme/#default-theme).

**Comment by Vane237** [comment]
Hey @helfi92, 
Is it possible to know where the tooltip was added to the Schema Table row seeing as the table is being generated and I tried looking at the code for the Schema Table and it is minified. Looking at the code for the tooltip that is added to the DOM(using the chrome debugger), the issue appears to be the styling(the top and left values are too big, as shown in the shared image)
<img width="591" alt="Screen Shot 2019-10-29 at 02 24 50" src="https://user-images.githubusercontent.com/55631241/67730158-7615fe00-f9f4-11e9-9653-f1814a7ab597.png">

About the z-index value, according to the material-ui documentation, its not advisable to change just one. I did manage to change the tooltip value but it still didn't resolve the issue.

**Comment by Vane237** [comment]
Hey @helfi92 ,
While debugging on the browser, I was actually able to make it display by adding a style to the popover and it displays the popover. I would like to know which style file i can include the popover style in.

<img width="1678" alt="Screen Shot 2019-10-31 at 12 20 14" src="https://user-images.githubusercontent.com/55631241/67956812-f99e3d80-fbf4-11e9-9335-9326dbcec960.png">

**Comment by helfi92** [comment]
Awesome! Feel free to put it in any file you think it makes sense. Here's how we typically add styles to our views/components: https://github.com/taskcluster/taskcluster/search?q=%22%40withStyles%22&unscoped_q=%22%40withStyles%22
